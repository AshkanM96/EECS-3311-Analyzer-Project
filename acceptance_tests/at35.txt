there_exists
type_check
evaluate
reset
reset
there_exists
open_set_enumeration
boolean_constant(false)
boolean_constant(false)
evaluate
type_check
boolean_constant(true)
boolean_constant(false)
boolean_constant(false)
boolean_constant(false)
close_set_enumeration
evaluate
reset
there_exists
open_set_enumeration
open_set_enumeration
boolean_constant(false)
close_set_enumeration
close_set_enumeration
evaluate
type_check
