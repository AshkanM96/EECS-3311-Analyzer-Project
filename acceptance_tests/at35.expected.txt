  Expression currently specified: ?
  Report: Expression is initialized.
->there_exists
  Expression currently specified: (|| ?)
  Report: OK.
->type_check
  Expression currently specified: (|| ?)
  Report: Error (Expression is not yet fully specified).
->evaluate
  Expression currently specified: (|| ?)
  Report: Error (Expression is not yet fully specified).
->reset
  Expression currently specified: ?
  Report: OK.
->reset
  Expression currently specified: ?
  Report: Error (Initial expression cannot be reset).
->there_exists
  Expression currently specified: (|| ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: (|| {?})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (|| {False, ?})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (|| {False, False, ?})
  Report: OK.
->evaluate
  Expression currently specified: (|| {False, False, ?})
  Report: Error (Expression is not yet fully specified).
->type_check
  Expression currently specified: (|| {False, False, ?})
  Report: Error (Expression is not yet fully specified).
->boolean_constant(True)
  Expression currently specified: (|| {False, False, True, ?})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (|| {False, False, True, False, ?})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (|| {False, False, True, False, False, ?})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (|| {False, False, True, False, False, False, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (|| {False, False, True, False, False, False})
  Report: OK.
->evaluate
  Expression currently specified: (|| {False, False, True, False, False, False})
  Report: True
->reset
  Expression currently specified: ?
  Report: OK.
->there_exists
  Expression currently specified: (|| ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: (|| {?})
  Report: OK.
->open_set_enumeration
  Expression currently specified: (|| {{?}})
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (|| {{False, ?}})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (|| {{False}, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (|| {{False}})
  Report: OK.
->evaluate
  Expression currently specified: (|| {{False}})
  Report: Error (Expression is not type-correct).
->type_check
  Expression currently specified: (|| {{False}})
  Report: (|| {{False}}) is not type-correct.
